### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by thet**
... as i work with it.

- Fix Py_DECREF typo in optimizations.c. (c compiler complained while building with buildout)
- Let buildout depend on cookiecutter to be able to create project skeletons.

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -15,6 +15,7 @@ recipe = zc.recipe.egg
 eggs =
     aioconsole
     guillotina
+    cookiecutter
```
Comment (thet) [inline buildout.cfg]: not sure, if setup.py should depend on it because setup.py also installs the console_scripts...

File: buildout.cfg [hunk 2]
```diff
@@ -62,6 +62,7 @@ eggs =
     pytest-cov
     pytest-watch
     guillotina [test]
+    cookiecutter
```
Comment (thet) [inline buildout.cfg]: not sure, if setup.py should depend on it because setup.py also installs the console_scripts...

**Comment by vangheem** [comment]
Thanks @thet! I'll fix the tests.

**Comment by vangheem** [comment]
Yah, I don't want it setup.py--it should be optional.

**Comment by vangheem** [comment]
Closing, going to fix with another PR. optimizations is another issue that we're aware of.

**Pull request closed without merging**